src/BaGet.Core.Server/Configuration/ConfigureForwardedHeadersOptions.cs

```diff
@@ -1,15 +1,31 @@
+using BaGet.Core;
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.HttpOverrides;
 using Microsoft.Extensions.Options;
+using System;
+using System.Collections.Generic;
 
 namespace BaGet.Configuration
 {
     public class ConfigureForwardedHeadersOptions : IConfigureOptions<ForwardedHeadersOptions>
     {
-        public void Configure(ForwardedHeadersOptions options)
+        private BaGetOptions bagetOptions;
+        public ConfigureForwardedHeadersOptions(IOptions<BaGetOptions> bagetOptions)
         {
-            options.ForwardedHeaders = ForwardedHeaders.XForwardedFor | ForwardedHeaders.XForwardedProto;
+            this.bagetOptions = bagetOptions.Value;
+        }
 
+        public void Configure(ForwardedHeadersOptions options)
+        {
+            if (!string.IsNullOrEmpty(bagetOptions.HostUrl))
+            {
+                Console.WriteLine("Host Url: " + bagetOptions.HostUrl);
```

Comment:
Can we remove this? Or do you think this will be useful for customers? If the latter, please log using an `ILogger<ConfigureForwardedHeaderOptions>` as not everyone has access to stdout to browse logs

Reply:
Mabe leaving it hast Debug message should be fine. If somehow problems occur you could enable debug and this might help.

I will take the logger framework. I wanted to change this before pull request but forgot it.